lib/src/configuration.cc

```diff
@@ -618,13 +606,22 @@ void Configuration::parseConfigFile()
                                     key, type_str) };
         };
 
+    bool broker_ws_uri_in_file = false;
+
     for (const auto& key : config_json.keys()) {
         const auto& opt_idx = defaults_.get<Option::ByName>().find(key);
 
-        if (opt_idx == defaults_.get<Option::ByName>().end())
-            throw Configuration::Error {
-                lth_loc::format("field '{1}' is not a valid configuration variable",
-                                key) };
+        if (opt_idx == defaults_.get<Option::ByName>().end()) {
+            if (key == "broker-ws-uris") {
+                check_key_type(key, "Array", lth_jc::DataType::Array);
```

Comment:
Perhaps we should add a comment about `broker-ws-uris` being a configfile-only option.